src/themes/rhdh/components.ts

```diff
@@ -454,7 +454,10 @@ export const components = (themeConfig: ThemeConfig): Components => {
     components.BackstageSidebar = {
       styleOverrides: {
         drawer: {
-          backgroundColor: general.sideBarBackgroundColor,
+          backgroundColor: general.sidebarBackgroundColor,
```

Comment:
I like that naming improvement, in general. But this might affect the compatibility, or was it never called `sideBarBackgroundColor` before?

Can we have maybe both types, and merge it here from both values? Prefer the new one, and take the old one as fallback?

```suggestion
          backgroundColor: general.sidebarBackgroundColor ?? general.sideBarBackgroundColor,
```